From dd8861eb67eb491e49c15095a0de7c8458c402ea Mon Sep 17 00:00:00 2001
From: Nicolas Le Bayon <nicolas.le.bayon@st.com>
Date: Mon, 5 Feb 2024 15:01:28 +0100
Subject: [PATCH 499/954] feat(stm32mp23-fdts): add fw-config file

This is a generic file to be used on all STM32MP23 boards, as what is
done for STM32MP25.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 1d33c594e7877f42ee43797bd2664a467f271ae6 feat(stm32mp23-fdts): add fw-config file]
Signed-off-by: Nicolas Le Bayon <nicolas.le.bayon@st.com>
Change-Id: I9d81b075db2eb674c54e33e590aa7fd31c85cee5
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/357549
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
---
 fdts/stm32mp23-fw-config.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)
 create mode 100644 fdts/stm32mp23-fw-config.dtsi

diff --git a/fdts/stm32mp23-fw-config.dtsi b/fdts/stm32mp23-fw-config.dtsi
new file mode 100644
index 0000000000..f1e4ab7ff6
--- /dev/null
+++ b/fdts/stm32mp23-fw-config.dtsi
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+/*
+ * Copyright (c) 2024, STMicroelectronics - All Rights Reserved
+ */
+
+#include "stm32mp25-fw-config.dtsi"
-- 
2.53.0

